tests/h/features/client_test.py

```diff
@@ -125,11 +133,13 @@ def fetcher(self):
             Feature(name='on-for-everyone', everyone=True),
             Feature(name='on-for-staff', staff=True),
             Feature(name='on-for-admins', admins=True),
+            Feature(name='on-for-cohort', cohorts=[FeatureCohort(name='cohort')])
         ])
 
     @pytest.fixture
     def req(self):
-        return DummyRequest(db=mock.sentinel.db_session)
+        return DummyRequest(db=mock.sentinel.db_session,
+                            authenticated_user='acct:foo@example.com')
```

Comment:
This is a bit weird, because `authenticated_user` isn't a string in the real application, it's a user object. I suspect you'll discover that this will need to change anyway if you address the comments above.